Test/TestDigestionMotif.cs

```diff
@@ -145,5 +145,28 @@ public static void TestSyntax()
                 Assert.Fail("Exception shold be thrown for incorrect syntax.");
             });
         }
+
+        [Test]
+        public static void TestMultiplePreventingCleavage()
+        {
+            var empty = new List<Modification>();
+            var digestionmotifs = DigestionMotif.ParseDigestionMotifsFromString("N[M]|,N[C]|,H[M]|,H[F]|");
```

Comment:
`N[M]|` or `N|[M]`? I thought it was the second one? or does it not matter?

Reply:
it doesnt matter